CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Unreleased
 
+BREAKING CHANGES:
+
+FEATURES:
+
+IMPROVEMENTS:
+* Remove `kubeVersion` in `Chart.yaml` since it was causing installs to fail on EKS and GKE [[GH-873](https://github.com/hashicorp/consul-helm/pull/873)]
+
+BUG FIXES:
+
+## 0.30.0 (March 18, 2021)
+
```

Comment:
I'll update this one in #871, so no need to add it! Sorry I forgot to change it during the release

Reply:
Ok will wait for that one to merge. Not sure how to pull in your changes on this PR.